### PR TITLE
Added simple graphql app

### PR DESCRIPTION
**Pull request by Arfey**
Sorry, for my estimate 😞

Now, i added structure for graphql api. It's chat because i want to show all possible of `GraphQl` (query, mutation, subscribes).

`contrib.py` - it's file that solve problem in few libs. I going to make pull request in these libs for remove that file in future.

todo:
 -- add correct ui (now ui has problems)
 -- add test
 -- add authorization

exapmle:
screen shot 2018-05-28 at 22 29 01

### CODE REVIEW

**Comment by jettify** [comment]
I suggest not to add authorization in this PR, it is already huge and requires some time to read.

File: demos/graphql/README.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pip install -r requirements-dev.txt
```
Comment (jettify) [inline demos/graphql/README.rst]: Probably good idea to update README early with information how to start app, as well as purpose of the app.

**Comment by Arfey** [comment]
Okay. I will add full `ui` and `readme`. I will wait for comments 😊

**Comment by jettify** [comment]
Lots of code, not sure I can review properly. But I do not mind merging if we have simple tests for most of handlers.

**Comment by webknjaz** [comment]
Also needs rebasing.

**Comment by webknjaz** [comment]
You probably didn't package it correctly and it didn't make it into dist.

**Comment by webknjaz** [comment]
Packaging it into an installable distribution requires knowledge of `setuptools` specifics (like manifest) and actual *need* of having it. (Yes, I'm still against putting `setup.py` every here and there).

**Comment by Arfey** [comment]
So, we have green build. ✅
I moved config files to package, also removed `pytest-asyncio` from my tests, because it's has conflict with tests in other projects. It's so bad. In future we should isolate requirements and run tests for different projects. 🤔

**Comment by jettify** [comment]
There are minor things here and there, but those if required can be addressed in separate PRs. As for packaging, I do not think that is big problem. We can address that later too.

**Pull request merged**